### PR TITLE
Feature/compose develop

### PR DESCRIPTION
**Pull request by jvinolas**
New docker-compose functionality for production and development:
- Production: docker-compose up / docker-compose build / docker-compose pull as usual
- Development docker-compose -f docker-compose.yml -f docker-compose.devel.yml [up|build]
The version is tagged in .env file. To override it (development) you can use export TAG=1.1.x

Also the base alpine-pandas image has been removed and all the other images are now independent.

### CODE REVIEW

File: dockers/app/Dockerfile [hunk 1]
```diff
@@ -1,27 +1,55 @@
...
+	curl \
+	openssh-client \
+	supervisor
+
+RUN apk add --no-cache --virtual .build_deps \
+	build-base \
+	python3-dev \
+	libffi-dev \
+	openssl-dev 
+RUN pip3 install pandas
```
Comment (NefixEstrada) [inline dockers/app/Dockerfile]: It should be `pip3 install --no-cache-dir`

File: dockers/app/Dockerfile [hunk 2]
```diff
@@ -1,27 +1,55 @@
...
 
-COPY dockers/app/certs.sh /
-COPY dockers/app/add-hypervisor.sh /
+CMD ["/usr/bin/supervisord", "-c", "/etc/supervisord.conf"]
 
-RUN mkdir /isard
-ADD ./src /isard
-RUN mv /isard/isard.conf.docker /isard/isard.conf
+FROM production as development
+RUN pip3 install ipython pytest
```
Comment (NefixEstrada) [inline dockers/app/Dockerfile]: It should be `pip3 install --no-cache-dir`

File: dockers/app/Dockerfile [hunk 3]
```diff
@@ -1,27 +1,55 @@
...
+RUN apk add --update bash vim openssh bash
```
Comment (NefixEstrada) [inline dockers/app/Dockerfile]: It should be `apk add --no-cache`

File: dockers/hypervisor/Dockerfile [hunk 4]
```diff
@@ -1,45 +1,53 @@
...
 
-RUN apk --no-cache add qemu-system-x86_64 libvirt netcat-openbsd libvirt-daemon dbus polkit qemu-img
 RUN ln -s /usr/bin/qemu-system-x86_64 /usr/bin/qemu-kvm
-RUN apk add openssh curl bash
-RUN ssh-keygen -A
-ADD dockers/hypervisor/reset-hyper.sh /
-RUN chmod 744 reset-hyper.sh
 
+# SSH configuration
 RUN echo "root:isard" |chpasswd
```
Comment (NefixEstrada) [inline dockers/hypervisor/Dockerfile]: I'm not sure if this line is required. We should test it

File: src/run_engine.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-from gevent.wsgi import WSGIServer
+from gevent.pywsgi import WSGIServer
```
Comment (NefixEstrada) [inline src/run_engine.py]: Should this really be changed? Which is the version that installs Alpine 3.8 by default? Maybe `pywsgi` is the "new version" of `wsgi` and it still exists in older versions before `wsgi` was deprecated

File: dockers/app/Dockerfile (hunk 1 above)
Comment (jvinolas) [reply dockers/app/Dockerfile]: Fixed

File: dockers/app/Dockerfile (hunk 2 above)
Comment (jvinolas) [reply dockers/app/Dockerfile]: fixed

File: dockers/app/Dockerfile (hunk 3 above)
Comment (jvinolas) [reply dockers/app/Dockerfile]: Fixed

File: dockers/hypervisor/Dockerfile (hunk 4 above)
Comment (jvinolas) [reply dockers/hypervisor/Dockerfile]: We leave this as it could be useful in development target

File: src/run_engine.py (hunk 5 above)
Comment (jvinolas) [reply src/run_engine.py]: It seems to work like that

**Pull request merged**